frontend/js/pages/DetailBattle.js

```diff
@@ -0,0 +1,90 @@
+import axios from 'axios';
+import React, { Component } from 'react';
+import styled from 'styled-components';
+
+import PokemonCard from '../components/PokemonCard';
+
+const StyledTitle = styled.span`
+  font-weight: 600;
+`;
+
+const StyledContainer = styled.div`
+  margin-top: 30px;
+`;
+
+const StyledRoundContainer = styled.div`
+  display: flex;
+  justify-content: space-evenly;
+  margin: 40px;
+`;
+const StyledVersus = styled.span`
+  font-weight: 700;
+  align-self: center;
+`;
+
+class DetailBattle extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      battle: {
+        winner: '',
+        id: 0,
+        creator_team: {
+          trainer: '',
+          team: [],
+        },
+        opponent_team: {
+          trainer: '',
+          team: [],
+        },
+      },
+    };
+  }
+
+  componentDidMount() {
+    // TODO: Will use pathname from props when I implement react-router-dom
+    const { pathname } = window.location;
+    const id = pathname.slice(pathname.lastIndexOf('/') + 1);
+    axios.get(`/api/battle/${id}`).then((response) => {
+      this.setState({ battle: response.data });
+      return response.data;
+    });
+  }
+
+  render() {
+    const { battle } = this.state;
+    return (
+      <>
+        <h1>Battle nº {battle.id}</h1>
+        <div>
+          <p>
+            <StyledTitle>Players: </StyledTitle>
+            {battle.creator_team.trainer} <span>VS</span> {battle.opponent_team.trainer}
+          </p>
+
+          <p>
+            <StyledTitle>Winner: </StyledTitle>
+            {battle.winner ? battle.winner : '?'}
+          </p>
+          <StyledContainer>
+            {battle.creator_team.team.map((pokemon, index) => (
+              <>
```

Comment:
Please convert this to a `div` and add a `key` prop to avoid this warning, [check here why it's important](https://reactjs.org/docs/lists-and-keys.html#keys).

![image](https://user-images.githubusercontent.com/23136832/80655588-72bc0780-8a55-11ea-8310-e0bacbdda01f.png)